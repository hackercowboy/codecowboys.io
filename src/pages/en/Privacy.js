import React, { Component } from 'react';
import { Container } from 'reactstrap';
import Markdown from 'react-markdown';
import { readFileSync } from 'fs';
import { join } from 'path';

import Content from '../../components/Content';
import Navigation from '../../components/Navigation';


const text = readFileSync(join(__dirname, './Privacy.md'), 'utf8');


export default class Privacy extends Component {
  render() {
    return (
      <Content space>
        <Navigation/>

        <Container className="article">
          <Markdown source={text} />
        </Container>
      </Content>
    );
  }
}

