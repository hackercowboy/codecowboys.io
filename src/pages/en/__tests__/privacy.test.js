import React from 'react';
import { shallow } from 'enzyme';

import Privacy from '../privacy';

describe('<Privacy/>', () => {
  it('should render', () => {
    const wrapper = shallow(<Privacy />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
