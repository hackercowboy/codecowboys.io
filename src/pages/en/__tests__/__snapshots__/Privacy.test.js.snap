// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Privacy/> should render 1`] = `
"<Main className={[undefined]}>
  <Header />
  <Container className=\\"article\\">
    <Privacy.md />
  </Container>
</Main>"
`;
